client/src/main/java/org/apache/falcon/client/FalconCLIException.java

```diff
@@ -27,7 +27,7 @@
 /**
  * Exception thrown by FalconClient.
  */
-public class FalconCLIException extends Exception {
+public class FalconCLIException extends RuntimeException{
```

Comment:
It will be worthwhile to add a 1 line comment on why it was done for future reference.